src/src/DataStructs/Modbus.cpp

```diff
@@ -118,7 +118,8 @@ bool Modbus::handle() {
         rxValue = rxValue << 8;
         char a = ModbusClient->read();
         rxValue    = rxValue | a;
-        LogString += ((int)a);  LogString += (" ");
+        LogString += static_cast<int>(a);  
+        LogString += (" ");
```

Comment:
This could be a char `' '`

Reply:
Ah yep, missed that one... or did I.... ;)
Thanks for the thorough check :)